test/functional/helpers/createFixture/clientScripts.js

```diff
@@ -0,0 +1,166 @@
+/*
+Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import path from "path";
+import fs from "fs";
+import { ClientFunction } from "testcafe";
+
+const edgeEnv = process.env.EDGE_ENV || "int";
+const alloyEnv = process.env.ALLOY_ENV || "int";
+
+// eslint-disable-next-line no-console
+console.log("EDGE ENV:", edgeEnv);
+// eslint-disable-next-line no-console
+console.log("ALLOY ENV:", alloyEnv);
+
+const localAlloyLibraryPath = path.join(
+  __dirname,
+  "../../../../dist/standalone/alloy.js"
+);
+
+let localAlloyCode;
+
+/**
+ * We're only testing against the alloy code found in /dist when alloyEnv
+ * is "int". Otherwise, we're testing against the alloy that's on the CDN
+ * and alloy code may not exist in /dist. This is why we don't always try to
+ * load the file from the file system.
+ */
+if (alloyEnv === "int") {
+  localAlloyCode = fs.readFileSync(localAlloyLibraryPath, "utf8");
+}
+
+const remoteAlloyLibraryUrl =
+  "https://cdn1.adoberesources.net/alloy/latest/alloy.js";
+
+const addVisitorLibraryClientScript = clientScripts => {
+  clientScripts.push({
+    content:
+      "document.write('<script src=\"https://github.com/Adobe-Marketing-Cloud/id-service/releases/latest/download/visitorapi.min.js\"></script>')"
+  });
+};
+
+const getFixtureClientScriptsForInt = options => {
+  const clientScripts = [];
+  const promisePolyfillPath = path.join(
+    __dirname,
+    "../promisePolyfill/promise-polyfill.min.js"
+  );
```

Comment:
We could just use the CDN-hosted polyfill like we do in `getFixtureClientScriptsForProd`, but it might be a bit slower because we would have to go over the internet to load it.